regulations/tests/views_preamble_tests.py

```diff
@@ -80,6 +91,27 @@ def test_get_integration(self, ApiReader, api_reader, CFRChangeToC):
             '4',
         )
 
+    @override_settings(
+        PREAMBLE_INTRO=make_intro('1', date.today() + timedelta(days=1)),
+    )
```

Comment:
Little consolation, but this should be a bit less ugly once we pull the data from the notice rather than the mock data.